package.json

```diff
@@ -27,24 +28,19 @@
   "peerDependencies": {
     "eslint-plugin-mocha": "^2.0.0",
     "eslint-plugin-react": "^4.1.0",
-    "eslint-plugin-json": "^1.2.0",
```

Comment:
These files should become dependencies instead of peer or devDependencies as our configuration extend these files. See https://github.com/godaddy/javascript/pull/31